### PR TITLE
Register to nickserv with a password

### LINKED ISSUE #203: bb-supy is not registered
bb-supy can't send to the #buildbot channel because it is not registered

chanserv won't +V it since it's not registered

I'm not sure how best to fix this.  I assume the channel's mode bits are set to require registration in order to minimize spam?  Can supybot send a password?

### PR DESCRIPTION
**Pull request by djmitche**
Fixes #203

I didn't test this at all, so I may have messed up the secret bit.

### CODE REVIEW

File: roles/supybot/templates/bb-supy.conf.j2 [hunk 1]
```diff
@@ -935,6 +935,103 @@ supybot.plugins.Owner.public: True
...
+
+###
+# Determines whether the bot will not join any channels until it is
+# identified. This may be useful, for instances, if you have a vhost
+# that isn't set until you're identified, or if you're joining +r
+# channels that won't allow you to join unless you identify.
+#
+# Default value: False
+###
+supybot.plugins.Services.noJoinsUntilIdentified: False
```
Comment (tardyp) [inline roles/supybot/templates/bb-supy.conf.j2]: looks like we want this to True..

File: roles/supybot/templates/bb-supy.conf.j2 [hunk 2]
```diff
@@ -935,6 +935,103 @@ supybot.plugins.Owner.public: True
...
+#
+# Default value: 
+###
+supybot.plugins.Services.NickServ: NickServ
+
+###
+# Determines what password the bot will use with NickServ when
+# identifying as bb-supy.
+#
+# Default value: raip5aeR7O
```
Comment (tardyp) [inline roles/supybot/templates/bb-supy.conf.j2]: that is not our passwd is it?

Comment (djmitche) [reply roles/supybot/templates/bb-supy.conf.j2]: It is.  Why did it get written there? :(

File: roles/supybot/templates/bb-supy.conf.j2 (hunk 1 above)
Comment (djmitche) [reply roles/supybot/templates/bb-supy.conf.j2]: Oo, good catch.

**Comment by djmitche** [comment]
OK, password changed.

**Pull request merged**